server/pkg/provision/auth.go

```
@@ -43,3 +61,119 @@ func emailSettings(ctx context.Context, s store.Storer) error {
 		})
 	})
 }
+
+// addAuthSuperUsers assigns BYPASS roles to users from AUTH_PROVISION_SUPER_USER value
+// When in Production, Corteza should stop and report an error.
+func addAuthSuperUsers(ctx context.Context, log *zap.Logger, s store.Storer, envOpt options.EnvironmentOpt) (err error) {
+	var (
+		val, has = os.LookupEnv("AUTH_PROVISION_SUPER_USER")
+	)
+
+	if envOpt.IsProduction() {
+		panic(errors.New(0, "Cannot provision super users in Production enviroment!"))
```

No need to panic here; we can just return the err